core/templates/pages/skill-editor-page/modal-templates/delete-misconception-modal.controller.spec.ts

```diff
@@ -87,14 +87,16 @@ describe('Delete Misconception Modal Controller', function() {
     });
   }));
 
-  it('should init the variables', function() {
-    expect($scope.skill).toEqual(skillObject);
-  });
+  it('should initialize $scope properties after controller is initialized',
+    function() {
+      expect($scope.skill).toEqual(skillObject);
+    });
 
-  it('should close the modal and get misconception on confirm', function() {
-    $scope.confirm();
-    expect($uibModalInstance.close).toHaveBeenCalledWith({
-      id: '2'
+  it('should close the modal and get misconception when confirming',
```

Comment:
I'm not sure about this... Can you explain what the code does?

Reply:
I changed the spec name to a better one.